docs/sections/contributor_guide/fork_pr_model.rst

```diff
@@ -1,4 +1,46 @@
+
+For future contributions, you may delete and then re-create your fork, or configure the official ``workflow-tools`` repo as `remote repository <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/configuring-a-remote-repository-for-a-fork>`_ on your clone and `sync upstream changes <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork>`_ to stay up-to-date with the official repo.
+
+Specifics for ``workflow-tools``
+--------------------------------
+
+When creating your PR, please follow these guidelines, specific to the ``workflow-tools`` project:
+
+* Ensure that your PR is targeting base repository ``ufs-community/workflow-tools`` and base branch ``develop``.
+* Complete the template that you will find the **Description** field pre-populated with. Provide an informative synopsis of your contribution, then mark appropriate checklist items by placing an "x" between their square brackets. You may tidy up the description by removing boilerplate text and non-selected checklist items.
+* Use the pull-down arrow on the green button below the description to initially create a `Draft pull request <https://github.blog/2019-02-14-introducing-draft-pull-requests/>`_.
+* Once your draft PR is open, visit its **Files changed** tab and add comments on any lines of code that you think reviewers will benefit from. Try to save time by proactively answering questions you suspect reviewers will ask.
+* Once your draft PR is marked up with your comments, return to the **Conversation** tab and click the **Ready for review** button.
+
+A default set of reviewers will automatically be added to your PR. You may add others, if appropriate. Reviewers may make comments, ask questions, or request changes on your PR. Respond to these as needed, making commits in your clone and pushing to your fork/branch. Your PR will automatically be updated when commits are pushed to its source branch in your fork, so reviewers will immediately see your updates.
+
+Merging
+-------
+
+Your PR is ready to merge when:
+
+#. It has been approved by a required number of ``workflow-tools`` core-developer reviewers.
+#. All conversations have been marked as resolved.
+#. All required checks have passed.
+
+These criteria and their current statuses are detailed in a section at the bottom of your PR's **Conversation** tab. Checks take some time to run, so please be patient.
+
+If you have write access to the ``workflow-tools`` repo, you may merge your PR yourself once the above conditions are met. If not, a ``workflow-tools`` core developer will perform the merge for you.
+
+Need Help?
+----------
+
+Please use comments in the **Conversation** tab of your PR to ask for help with any difficulties you encounter using this procedure!
```

Comment:
Overall, it looks good to me. I do have one slight hang-up here, though, since a contributor struggling with following these steps may not have been able to successfully create a PR to leave comments in.

Reply:
That's a good point. I don't think there's anything `workflow-tools`-specific that would stand in the way of at least getting a PR open, so hopefully existing help resources from GitHub, StackOverflow, etc. would get people that far. I was thinking that we'd be fielding questions more about why `pylint` or `mypy` fail, or about getting 100% unit-test coverage, or other issues more related to `workflow-tools` or our code-quality tooling.

Should we suggest opening a GitHub Issue in `workflow-tools` as an alternative, something like
> If you encounter difficulties using this procedure, please request help via a comment in your PR, or open an issue in the `workflow-tools` GitHub repo.

Reply:
I'm not sure it's necessary, the more I think about it. Perhaps it serves as a weed-out test. I think this, as well as the `[skip ci]` issue, relate back to the idea that we shouldn't be teaching people how to use git or GitHub.